models/repo_unit.go

```diff
@@ -16,7 +16,7 @@ import (
 
 // RepoUnit describes all units of a repository
 type RepoUnit struct {
-	ID          int64
+	ID          int64           `xorm:"pk autoincr"`
```

Comment:
This is unnecessary.

Reply:
You're right. I found my issue with this was inserting units already with ids. Thanks for reminding me this isn't needed

Reply:
Removed.